examples/tk-gas-station/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@turnkey/example-gassy-station",
```

Comment:
an old relic :') let's rename this to the less fun `@turnkey/example-tk-gas-station`